src/main/scala/mesosphere/chaos/http/HttpConf.scala

```diff
@@ -40,4 +50,14 @@ trait HttpConf extends ScallopConf {
     // Default to the asset path in the jar
     case _                  => getClass.getClassLoader.getResource("assets")
   }
+
+  private lazy val httpCredentialsEnvName: String = "MESOSPHERE_HTTP_CREDENTIALS"
```

Comment:
Mind moving these constants to a companion object and removing the `lazy` modifier?